frontend/src/entities/configuration/deployment-configuration.tsx

```diff
@@ -123,31 +122,34 @@ export function DeploymentConfiguration () {
             }
```

Comment:
Do we still need this header and top level container? I think it makes sense to have 
```
<>
    <Container></Container>
    <Container></Container>
</>
```
Rather than
```
<Container>
    <Container></Container>
    <Container></Container>
</Container>
```

This approach seems to match the other tabs

Reply:
I asked Cindy this when we were going through the text and she said she liked the current layout. All the tabs we have currently look identical.